joeynmt/training.py

```diff
@@ -461,16 +476,16 @@ def _log_examples(self, sources: List[str], hypotheses: List[str],
         for p in self.log_valid_sents:
             if p >= len(sources):
                 continue
-            self.logger.debug("Example #%d", p)
+            self.logger.info("Example #%d", p)
             if sources_raw is not None:
-                self.logger.debug("\tRaw source: %s", sources_raw[p])
-            self.logger.debug("\tSource: %s", sources[p])
+                self.logger.info("\tRaw source: %s", sources_raw[p])
+            self.logger.info("\tSource: %s", sources[p])
             if references_raw is not None:
-                self.logger.debug("\tRaw reference: %s", references_raw[p])
-            self.logger.debug("\tReference: %s", references[p])
+                self.logger.info("\tRaw reference: %s", references_raw[p])
+            self.logger.info("\tReference: %s", references[p])
             if hypotheses_raw is not None:
-                self.logger.debug("\tRaw hypothesis: %s", hypotheses_raw[p])
-            self.logger.debug("\tHypothesis: %s", hypotheses[p])
+                self.logger.info("\tRaw hypothesis: %s", hypotheses_raw[p])
+            self.logger.info("\tHypothesis: %s", hypotheses[p])
```

Comment:
Why the shift from debug to info?

Reply:
Because it doesn't show up otherwise ;-)

Reply:
see log file ;)